examples/ui.content/src/content/jcr_root/conf/core-components-examples/_sling_configs/.content.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0"
+    jcr:primaryType="nt:unstructured">
+    <com.adobe.cq.wcm.core.components.internal.DataLayerConfig
+        jcr:primaryType="nt:unstructured"
+        dataLayerEnabled="{Boolean}true"/>
```

Comment:
Do we need to have a name this long? Since it's a property of `com.adobe.cq.wcm.core.components.internal.DataLayerConfig`, I would just have it as `enabled="{Boolean}true"`. I would also shorten the field name in the config class.

Reply:
addressed in https://github.com/adobe/aem-core-wcm-components/pull/991/commits/e2f70b341d712c9dd8b7cfa8af9249b3cf85e539